.pre-commit-config.yaml

```diff
@@ -84,9 +84,10 @@ repos:
     exclude: ".*localized.*"
 
 - repo: https://github.com/dotnet/format
-  rev: "7e343070a0355c86f72bdee226b5e19ffcbac931"  # TODO - update to a tagged version when one that includes the hook is ready.
+  rev: v5.1.225507
   hooks:
   - id: dotnet-format
+    entry: dotnet-format whitespace
```

Comment:
can you add a line saying we added this because style and analyzer doesn't support --folder?

Reply:
Discussed offline -- this may or may not be true in the future and context should be in the git history so this may be unnecessary to do.